syntax/lsp-hover.vim

```diff
@@ -41,12 +41,11 @@ function! s:cleanup_markdown() abort
     " Workaround for: https://github.com/palantir/python-language-server/issues/386
     if has('conceal')
         for l:escaped_char in ['`', '*', '_', '{', '}', '(', ')', '<', '>', '#', '+', '.', '!', '-']
-            execute printf('syntax match markdownEscape "\\[][%s]" conceal cchar=%s', l:escaped_char, l:escaped_char)
+            execute printf('syntax region vimLspMarkdownEscape matchgroup=Conceal start="\\\ze[%s]" end="[%s]\zs" concealends', l:escaped_char, l:escaped_char)
         endfor
     end
 
-    " Don't highlight concealed chars
-    highlight clear Conceal
+    highlight link vimLspMarkdownEscape Pmenu
```

Comment:
I'm using a custom background for `LspHover`s, so with this change, it looks like this:
![image](https://user-images.githubusercontent.com/10748726/74520298-8a382980-4f17-11ea-9554-e5c3767003a2.png)

Instead of clearing `Conceal`, maybe you can add a new highlight group `LspConceal` that is cleared instead?

Reply:
Is it only the background issue? or a italic?

Reply:
Only the background. Italic is because pyls is not escaping the `_`.

Reply:
How did you set up LspHover background?

Reply:
I have added it to the docs here: https://github.com/prabirshrestha/vim-lsp/blob/master/doc/vim-lsp.txt#L264-L279

Reply:
1. So, we shouldn't globally clear `Conceal` in any case.
2. Current implementation uses `cchar` which uses `Conceal` highlight group
3. Is there a way to query vim/nvim popup window for highlighting it uses? To be able to use it instead of `Pmenu`?

Reply:
Huh! It is easy -- we don't have to define highlighting for the `vimLspMarkdownEscape`!

Reply:
Getting the option `'wincolor'` or `'winhighlight'` should work. But maybe it's easier to define another highlight group without properties, so that it's "transparent"?

Reply:
Exactly :smile:

Reply:
Could you try again (I have force pushed changes)